signalfx-yammer/src/main/java/com/signalfx/codahale/reporter/MetricMetadata.java

```diff
@@ -0,0 +1,74 @@
+package com.signalfx.codahale.reporter;
```

Comment:
This file looks nearly if not identical to https://github.com/signalfx/signalfx-java/blob/master/signalfx-codahale/src/main/java/com/signalfx/codahale/reporter/MetricMetadata.java can you refactor this please?